src/memory.rs

```diff
@@ -127,7 +127,102 @@ pub fn to_java_heap_value(
     }
 }
 
+/// Parsed representation of a K8s memory/storage resource limit.
+#[derive(Clone, Copy, Debug)]
+pub struct MemoryQuantity {
+    pub value: f32,
+    pub unit: BinaryMultiple,
+}
+
 impl MemoryQuantity {
+    pub fn from_gibi(gibi: f32) -> Self {
+        Self {
+            value: gibi,
+            unit: BinaryMultiple::Gibi,
+        }
+    }
+
+    pub fn from_mebi(mebi: f32) -> Self {
+        Self {
+            value: mebi,
+            unit: BinaryMultiple::Mebi,
+        }
+    }
```

Comment:
I do not really see the point of this. I would combine this into a generic Self::new(value: f32, unit: ...) or use MemoryQuantity {} directly.

Reply:
I don't want to expose the `BinaryMultiple` if I can avoid it. It also reads nicely in code. If you don't oppose strongly, I'd like to keep it.

Reply:
BinaryMultiple is pub and exposed? What if we need kibi etc.? Not sure what the gain is here.